core/config/docs/core.toml

```diff
@@ -587,13 +587,17 @@ DisableRateLimiting = false # Default
 # Enabled turns trace collection on or off. On requires an OTEL Tracing Collector.
 Enabled = false # Default
 # CollectorTarget is the logical address of the OTEL Tracing Collector.
-CollectorTarget = "localhost:4317" # Example
+CollectorTarget = 'localhost:4317' # Example
 # NodeID is an unique name for this node relative to any other node traces are collected for.
-NodeID = "NodeID" # Example
+NodeID = 'NodeID' # Example
 # SamplingRatio is the ratio of traces to sample for this node.
 SamplingRatio = 1.0 # Example
+# Mode is a string value. 'secure' or 'insecure' are the only values allowed. If set to 'insecure', TLSCertPath can be unset, meaning traces will be sent over plaintext to the collector.
```

Comment:
can be unset or must be unset? iiuc the code errors if insecure && path is set

Reply:
fwiw, i think the ux is simpler if the behavior is 
if insecure => ignore the path
if secure => validate and use the path

Reply:
It's feasible that there are additional modes, like using a token in conjuction with TLS, a custom auth strategy, maybe something to do with using a cloud provider.

Reply:
hmm - if we want to keep the door open to additional modes, is 'tls' a better substitute for 'secure'?